dynosaur_derive/src/lib.rs

```diff
@@ -139,40 +139,110 @@ impl Parse for Bridge {
 ///
 /// The `Dyn` struct produced by this macro has the following constructors:
 ///
-/// ```ignore
+/// ```
+/// # struct DynTrait<'a>(&'a i32);
+/// # trait Trait {}
 /// impl<'a> DynTrait<'a> {
-///     fn new_box(from: impl Trait) -> Box<Self> { ... }
-///     fn new_arc(from: impl Trait) -> std::sync::Arc<Self> { ... }
-///     fn new_rc(from: impl Trait) -> std::rc::Rc<Self> { ... }
-///     fn from_ref(from: &'a impl Trait) -> &'a Self { ... }
-///     fn from_mut(from: &'a mut impl Trait) -> &'a mut Self { ... }
+///     fn new_box(from: impl Trait) -> Box<Self> { todo!() }
+///     fn new_arc(from: impl Trait) -> std::sync::Arc<Self> { todo!() }
+///     fn new_rc(from: impl Trait) -> std::rc::Rc<Self> { todo!() }
+///     fn from_ref(from: &'a impl Trait) -> &'a Self { todo!() }
+///     fn from_mut(from: &'a mut impl Trait) -> &'a mut Self { todo!() }
 /// }
 /// ```
 ///
 /// Normally a concrete type behind a pointer coerces to `dyn Trait` implicitly.
 /// When using the `Dyn` struct created by this macro, such conversions must be
 /// done explicitly with the provided constructors.
 ///
+/// ## Bridge impls
+///
+/// Dynosaur writes the following *bridge implementations* for your trait:
+///
+/// ```
```

Comment:
Don't we want to place `#[dynosaur::dynosaur(DynTrait)]` here?

Reply:
This line is just to make the example compile. It's hidden from the docs. If I added the attribute it would create overlapping impls.